bindings/python/eblob_python.cpp

```diff
@@ -72,13 +73,18 @@ struct eblob_py_iterator : eblob_iterate_control, boost::python::wrapper<eblob_i
 	}
 
 	static int iterator(struct eblob_disk_control *dc, struct eblob_ram_control *rc __attribute__((unused)),
-			void *data, void *priv, void *thread_priv __attribute__((unused)))
+			    int fd, uint64_t data_offset, void *priv, void *thread_priv __attribute__((unused)))
 	{
+		std::auto_ptr<char> p(new char[dc->data_size]);
+
+		::ssize_t bytes = pread(fd, p.get(), dc->data_size, data_offset);
+		if (bytes != dc->data_size)
+			return 0;
```

Comment:
Why does it return `0` if it failed to read whole data at once? May be we can use `__eblob_read_ll` here?